src/cdk/v2/destinations/koala/procWorkflow.yaml

```diff
@@ -29,7 +29,8 @@ steps:
         identifies: [{
           type: $.context.messageType,
           sent_at: .message.().({{{{$.getGenericPaths("timestamp")}}}}),
-          traits: koTraits 
+          traits: koTraits,
+          source: "rudderstack-cloud"
```

Comment:
Shouldn't we first check if the source was already provided? we shouldn't be overriding if provided already IMO. @gustavowt @ItsSudip

Reply:
In the context of koala, that would give us false information. This is a `protected` attribute for us and should not be overridden since we use that to identify the source during the identitfy call.
Note that this is only set on the identify call so that it can be used on the tracking call still.